devel/phosg/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.1
+PortGroup               github 1.0
```

Comment:
Would you mind adding the following (or something similar) to both this and the other port? (see [wiki](https://guide.macports.org/#reference.portgroup.github.distfilestrategy))

```tcl
github.tarball_from archive
```

By default, the tar is fetched from `https://codeload.github.com/fuzziqersoftware/phosg/legacy.tar.gz/e1bd000?dummy=`, which is a legacy method.

Note that this would then require the checksums to be changed. Run `sudo port clean --all NAME` to remove the old downloaded files.